inksdk/inksample/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.inksample">
```

Comment:
Better to change the package name to a msft one. Example: https://github.com/microsoft/surface-duo-sdk/blob/main/bottomnavigation/sample/src/main/AndroidManifest.xml